src/components/Steam.vue

```diff
@@ -122,13 +124,23 @@ export default Vue.extend({
 <style scoped lang="scss">
 .title {
   max-height: 48px;
-  overflow: hidden;
   padding: 10px;
+  overflow: hidden;
+  overflow-x: scroll;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+  &::-webkit-scrollbar {
+    height: 0;
+  }
+  &-text {
+    white-space: nowrap;
+    padding-right: 10px;
+  }
 }
 .fixed {
   position: fixed;
   // TODO Find out how to contain a position:fixed element in parent bounds
-  width: calc(100vw - 58%);
+  width: calc(50% - 24px);
```

Comment:
Out of curiosity, is there anything wrong with using `rem` everywhere (the current way I'm doing it)?

Reply:
I should probably start using `rem`. I just haven't been and don't have a good intuition of how it scales for margins and padding. In this case the 24px is 2x the default `v-container` padding